akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala

```diff
@@ -144,11 +149,21 @@ private[akka] class Shard(
   var state = State.Empty
   var idByRef = Map.empty[ActorRef, EntityId]
   var refById = Map.empty[EntityId, ActorRef]
+  var lastMessageTimestamp = Map.empty[EntityId, Long]
   var passivating = Set.empty[ActorRef]
   val messageBuffers = new MessageBufferMap[EntityId]
 
   var handOffStopper: Option[ActorRef] = None
 
+  import context.dispatcher
+  val passivateIdleTask = if (settings.passivateIdleEntityAfter.toNanos > 0) {
+    val idleInterval = 1.second // FIXME
+    log.info("Idle entities will be passivated after {}", PrettyDuration.format(settings.passivateIdleEntityAfter))
```

Comment:
[{}]

Reply:
Also didn't think it quite through, now it's logged once for every started shard. Should be logged from the region rather.